### PR TITLE
Control when an attempt is made to upload releasenotes.md

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by gep13** [comment]
@pascalberger @asbjornu let me know what you think of this...

**Comment by pascalberger** [comment]
Thanks @gep13! LGTM

**Comment by gep13** [comment]
And success: https://ci.appveyor.com/project/GitTools/gitversion/build/4.0.0-PullRequest1093.1141.build.1092

**Comment by gep13** [comment]
Now just to wait a few extra minutes for Travis 😄

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+                "${workspaceRoot}/build.cake",
+                "--debug",
+                "--verbosity=diagnostic"
+            ],
+            "cwd": "${workspaceRoot}",
+            "stopAtEntry": true,
+            "externalConsole": false
+        }
+    ]
+}
```
Comment (asbjornu) [inline .vscode/launch.json]: Should anything in the `.vscode` folder be committed?

Comment (gep13) [inline .vscode/launch.json]: @asbjornu I wasn't aware of any rule that it shouldn't be. That file is an easy way to share the required launch command for debugging build.cake file in VSCode.

Comment (pascalberger) [inline .vscode/launch.json]: Yes, these are the settings for debugging and should be committed (one of the three file: `settings.json`, `tasks.json` and `launch.json` which should be under version control)

Comment (pascalberger) [inline .vscode/launch.json]: @asbjornu See also [here](https://www.gitignore.io/api/visualstudiocode), where they are also explicitely excluded in the `.gitignore` file

Comment (gep13) [inline .vscode/launch.json]: @pascalberger nice, I can add that to the gitignore file.  Just away to sign off for the night though, so if Travis ever finishes, and we are all happy, feel free to merge, and I can fix that later.

Comment (pascalberger) [inline .vscode/launch.json]: @gep13  Unfortunately a test case, not releated to the release notes, failed on OS X: https://travis-ci.org/GitTools/GitVersion/jobs/177131223.

Comment (asbjornu) [inline .vscode/launch.json]: @gep13, @pascalberger: Okay, good to know. I agree that it should be in the `.gitignore` just to make it explicit. 🙂

The `set_PriorityClass` problem is hopefully fixed by GitTools/GitTools.Core#38, but we need to get that released somehow. Kind of related to this PR, but for different reasons and another repository. 😄

**Comment by gep13** [comment]
Relates to #1088

**Pull request merged**